src/styles/blocks/info.scss

```
@@ -0,0 +1,63 @@
+.main__info {
```

```suggestion
.main__info {
```
'main__info' is an element of block `main` (as well as `main__contacts`). These styles should be n another file. Nove them to `main.scss`. Or it would be better to create a new file `main.scss` for the `main` block and rename your current `main.scss` to `styles.scss`